assets/javascripts/discourse/connectors/user-card-location-and-website/replace-location.js.es6

```diff
@@ -0,0 +1,13 @@
+export default {
+  setupComponent(args, component) {
+    const enabled = Discourse.SiteSettings.location_users_map;
+    if (enabled) {
```

Comment:
This condition should match the template condition to insert the ``map-location`` element. Make a single property which determines if the site setting is on and the geo_location property is not empty and use that in both the template and for this conditional.

Reply:
Fixed by this 1923818b7415f9632946c8f3fab1d81d3afb81c7